content/300-guides/050-database/100-developing-with-prisma-migrate/100-advanced-migrate-scenarios.mdx

```diff
@@ -94,6 +94,61 @@ To rename the `biograpy` field to `biography`:
 
 You can use the same technique to rename a `model` - edit the generated SQL to _rename_ the table rather than drop and re-create it.
 
+## Using the expand and contract pattern to evolve the schema without downtime
+
+One of the challenges when renaming fields is that it can lead to downtime in the time frame between applying the rename migration and deloying a new version of the application code which uses the renamed field.
+
+This can be mitigated by breaking down the steps required to rename a field into a series of discrete steps designed to introduce the change gradually. This pattern is known as the _expand and contract pattern_.
+
+The pattern takes two components into consideration: your application code accessing the database, and the database schema which you intend to alter.
+
+With the _expand and contract_ pattern, renaming the field `bio` to `biography` would look as follows with Prisma:
+
+1. Add the new `biography` field to your Prisma schema and create a migration.
```

Comment:
It's probably best to be consistent in whether these steps end with a period or not.